README.md

```diff
@@ -163,6 +163,19 @@ See the full [tutorial](docs/tutorial.md) for more details.
   build/sbt generate
   ``` 
   This will regenerate the OpenAPI data models in the UC server and data models + APIs in the client SDK.
+
+
+## Setting up IDE
```

Comment:
I will merge this PR but I think this section needs basic instructions on how to set up IntelliJ / Eclipse. Code formatting is the not the only thing in the IDE set up.. is it?

Reply:
can you improve it in a follow up PR?

Reply:
@tdas just to confirm: Installing the `google-java-format` plugin is really the only step required and https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides contains all the steps required to install + enable it. I figured it should be fine to refer to the original docs for this rather than try and duplicate something here which might eventually get outdated.
Let me know if you still think that we should add some more text here

Reply:
The codestyle part is fine. But since we are making a general section on "Setting up IDE", I think its good to say how to open up the project in IntelliJ or Eclipse. Just a simple thing like load the build.sbt (at least for IntelliJ that I use) is sufficient.

Reply:
I've opened https://github.com/unitycatalog/unitycatalog/pull/170 to address that